.bazelrc

```diff
@@ -26,6 +26,9 @@ build --heap_dump_on_oom
 # Speed up all builds by not checking if output files have been modified.
 build --noexperimental_check_output_files
 
+# This is a mandatory flag.
+build --@rules_python//python/config_settings:python_version=3.10
```

Comment:
If you add the `default = True` attribute to the python toolchain in the ` MODULE` file, then you don't need this, right?

Reply:
Didn't change the toolchain in rules_ros. The flag isn't needed in rules_ros but it's needed in the examples repo.

Reply:
Why is this required in the examples repo?

Reply:
Otherwise Bazel can't figure out that pip deps are indeed for Python 3.10 toolchain. E.g. I get:

```
ERROR: /home/user/.cache/bazel/_bazel_user/6e0e647a7496b7382203f7000f7beb27/external/rules_python~~pip~rules_ros_pip_deps/empy/BUILD.bazel:27:6: configurable attribute "actual" in @@rules_python~~pip~rules_ros_pip_deps//empy:pkg doesn't match this configuration: No matching wheel for current configuration's Python version.

The current build configuration's Python version doesn't match any of the Python
wheels available for this wheel. This wheel supports the following Python
configuration settings:
    //_config:is_python_3.10

To determine the current configuration's Python version, run:
    `bazel config <config id>` (shown further below)
and look for
    rules_python//python/config_settings:python_version

If the value is missing, then the "default" Python version is being used,
which has a "null" version value and will not match version constraints.


This instance of @@rules_python~~pip~rules_ros_pip_deps//empy:pkg has configuration identifier aaa66e7. To inspect its configuration, run: bazel config aaa66e7.

For more help, see https://bazel.build/docs/configurable-attributes#faq-select-choose-condition.

ERROR: Analysis of target '//chatter:talker_tests' failed; build aborted: Analysis failed
```

Reply:
If this is required for the examples repo only, would it make sense to have a `.bazelrc` for the examples repo only to avoid this for the main repo?

Reply:
I don't want to maintain two .bazelrc files, that's why I synmlink-ed the rules_ros2 repo one to the examples repo. The flag doesn't harm the main repo.